### ISSUE TITLE
validation loss

### ISSUE THREAD

**Issue by Tangzixia** [Joker316701882/Deep-Image-Matting#39]
validation loss is very high about 109, however training loss is 0.5, so can you help me with this issue?

**Comment by ssxlconch** [comment]
> validation loss is very high about 109, however training loss is 0.5, so can you help me with this issue?

sorry for bothering
do you have the pretrained model? could you please share to me?
my email:2741592109@qq.com
